spec/requests/pull_requests/time_to_merge_prs/index_spec.rb

```diff
@@ -89,7 +89,7 @@
 
     it 'returns data only for all repositories' do
       assigns(:pull_requests).each do |_, pull_request_url|
-        expect(all_pull_requests_url).to include(pull_request_url.first)
+        expect(all_pull_requests_url).to include(pull_request_url.first[:html_url])
```

Comment:
what about validating the new _value_ added?

Reply:
The value already existed. It is validated in "Events PullRequest Spec"